x-pack/plugins/security_solution/public/management/links.ts

```diff
@@ -79,33 +103,80 @@ export const links: LinkItem = {
     {
       id: SecurityPageName.policies,
       title: POLICIES,
+      description: i18n.translate('xpack.securitySolution.appLinks.policiesDescription', {
+        defaultMessage:
+          'Use policies to customize endpoint and cloud workload protections and other configurations.',
+      }),
+      landingIcon: IconEndpointPolicies,
       path: POLICIES_PATH,
       skipUrlState: true,
       experimentalKey: 'policyListEnabled',
     },
     {
       id: SecurityPageName.trustedApps,
       title: TRUSTED_APPLICATIONS,
+      description: i18n.translate(
+        'xpack.securitySolution.appLinks.trustedApplicationsDescription',
+        {
+          defaultMessage:
+            'Improve performance or alleviate conflicts with other applications running on your hosts.',
+        }
+      ),
+      landingIcon: IconTrustedApplications,
       path: TRUSTED_APPS_PATH,
       skipUrlState: true,
     },
     {
       id: SecurityPageName.eventFilters,
       title: EVENT_FILTERS,
+      description: i18n.translate('xpack.securitySolution.appLinks.eventFiltersDescription', {
+        defaultMessage: 'Exclude unwanted applications from running on your hosts.',
+      }),
+      landingIcon: IconEventFilters,
       path: EVENT_FILTERS_PATH,
       skipUrlState: true,
     },
     {
       id: SecurityPageName.hostIsolationExceptions,
       title: HOST_ISOLATION_EXCEPTIONS,
+      description: i18n.translate('xpack.securitySolution.appLinks.hostIsolationDescription', {
+        defaultMessage: 'Allow isolated hosts to communicate with specific IPs.',
+      }),
+      landingIcon: IconHostIsolation,
       path: HOST_ISOLATION_EXCEPTIONS_PATH,
       skipUrlState: true,
     },
     {
       id: SecurityPageName.blocklist,
       title: BLOCKLIST,
+      description: i18n.translate('xpack.securitySolution.appLinks.blocklistDescription', {
+        defaultMessage: 'Exclude high volume or unwanted events being written into Elasticsearch.',
```

Comment:
This message is for event filters

Reply:
Good catch! I completely missed it. I followed the design and ignored the details.

<img width="895" alt="Screenshot 2022-05-17 at 10 09 27" src="https://user-images.githubusercontent.com/1490444/168762338-20f29207-6aaa-471c-8f21-b6e7cbd22ee3.png">